Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazeHidUsages.h

```diff
@@ -0,0 +1,59 @@
+//Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+//See LICENSE in the project root for license information.
+
+#pragma once
+
+BEGIN_NAMESPACE_GAZE_INPUT
+
+/// <summary>
+/// This enum specifies the various HID usages specified by the EyeHeadTracker HID specification
+///
+/// http://www.usb.org/developers/hidpage/HUTRR74_-_Usage_Page_for_Head_and_Eye_Trackers.pdf
```

Comment:
This pdf gives me a 404

Reply:
Fixed. usb.org did a massive overhaul of their web presence, so the urls were recently changed.